contents/handbook/who-we-are-building-for.md

```diff
@@ -21,8 +21,8 @@ Why? We believe that the best tech companies are increasingly engineering-led. B
 | **Description** | Startups that have product-market fit and are quickly scaling up with new customers, hiring, and adding more revenue. |
 | **Criteria** | - 15-500 employees<br />- $100k+/month in revenue _or_ very large number of consumer users<br />- Raised from leading investors<br />- Not yet IPO'ed |
 | **Why they matter?** | - Able to efficiently monetize them<br />- Very quick sales cycle<br />- Act as key opinion leaders for earlier-stage startups/slower moving companies<br />- Strong opinions on what they need - helping us build a better product |
-| **Job role** | We build for the power users of the **product team**<br /><br />**Primary focus**<br />- [Product engineers](/blog/what-is-a-product-engineer)<br/>- Technical founders <br />- Highly technical product managers <br /><br />**Should be usable by**:<br />- Designers<br />- Less technical product managers<br />- Marketers<br />|
-| **Examples** | PostHog anytime from their Series B to IPO, Linear, Ramp, Vercel, Retool |
+| **Job role** | We build for engineers<br /><br />**Primary focus**<br />- [Product engineers](/blog/what-is-a-product-engineer)<br/>- Technical founders <br />- Highly technical product managers <br /><br />**Should be usable by**:<br />- Designers<br />- Less technical product managers<br />- Data engineers<br />- Marketers<br />|
```

Comment:
This is somewhat vague. What does it mean to be "usable by data engineers"? Does that mean build features for them, only if it's in _service of_ product engineers? Does that mean they should simply be able to log in?

I think this might be best if we just removed it. We're focused on product engineers, period. We can build features that others ask for, but only if product engineers would use it and are asking for it also.

Reply:
I've left a comment above that clarifies this a bit (different section, where "usable" is also mentioned.